Gemfile

```diff
@@ -9,29 +9,19 @@ gem 'pg', '~> 1.1'
 
 gem 'puma', '~> 5.0'
 
-group :development, :test do
-  gem 'byebug', platforms: %i[mri mingw x64_mingw]
-end
-
 group :development do
   gem 'listen', '~> 3.3'
 end
 
-group :test do
-  # Adds support for Capybara system testing and selenium driver
-  gem 'capybara', '>= 3.26'
-  gem 'selenium-webdriver'
-  # Easy installation and use of web drivers to run system tests with browsers
-  gem 'webdrivers'
-end
-
 group :development, :test do
   gem 'brakeman'
   gem 'bundler-audit', '~> 0.8.0'
+  gem 'byebug', platforms: %i[mri mingw x64_mingw]
   gem 'fasterer'
   gem 'overcommit'
   gem 'rails_best_practices'
   gem 'reek'
+  gem 'rspec-rails', '~> 5.0.0'
```

Comment:
Good point to define all test stack in this pr in test block: json_matchers, shoulda-matchers, simplecov, simplecov-lcov, undercover